### PR TITLE
Add conflict patch for Pilgrim's Delight and Majestic Mountains.

### PR DESCRIPTION
**Pull request by jgostling**
Added both clean CRC for the patch file, and patching conditions for the original mod.

### CODE REVIEW

File: masterlist.yaml [hunk 1]
```diff
@@ -16720,3 +16720,22 @@ plugins:
           - 'Whiterun Expansion SSE - NON Open Cities Version'
           - '[Bells of Skyrim Compatibility Patches](https://www.nexusmods.com/skyrimspecialedition/mods/34472/)'
         condition: 'active("WhiterunAddition.esp") and not active("Bells of Skyrim-Whiterun Expansion Patchs.esp")'
+
+  ### Majestic Mountains - Pilgrims Delight Patch ###
+  - name: MM_PD_Patch.esp
```
Comment (MacSplody) [inline masterlist.yaml]: File name names should be wrapped in single quotes.
```suggestion
  - name: 'MM_PD_Patch.esp'
```

File: masterlist.yaml [hunk 2]
```diff
@@ -16720,3 +16720,22 @@ plugins:
...
+    url:
+      - 'https://www.nexusmods.com/skyrimspecialedition/mods/35107'
+    clean:
+      - crc: 0x97709110
+        util: 'SSEEdit v4.0.2'
+
+  ### Pilgrim's Delight - High Hrothgar Shrine Retextures ###
+  - name: PilgrimsDelight.esp
```
Comment (MacSplody) [inline masterlist.yaml]:
```suggestion
  - name: 'PilgrimsDelight.esp'
```

**Review by MacSplody — CHANGES_REQUESTED** [review]
The rest looks fine, it just needs the single quotes.

File: masterlist.yaml [hunk 3]
```diff
@@ -16720,3 +16720,22 @@ plugins:
...
+    url:
+      - 'https://www.nexusmods.com/skyrimspecialedition/mods/3273/'
+    msg:
+      - <<: *patch3rdParty
+        subs:
+          - Majestic Mountains - Cathedral Concept
```
Comment (MacSplody) [inline masterlist.yaml]: List entries need the quotes as well.
```suggestion
          - 'Majestic Mountains - Cathedral Concept'
```

**Comment by jgostling** [comment]
Sorry if this is a dumb question, but... Do I need to do something else beyond committing the suggested changes?

**Comment by MacSplody** [comment]
> 
> 
> Sorry if this is a dumb question, but... Do I need to do something else beyond committing the suggested changes?

Nope that's fine.

**Review by MacSplody — APPROVED** [review]

**Comment by MacSplody** [comment]
@jgostling Thanks.

**Pull request merged**